### DISCUSSION TITLE
Issues forms: control textarea size

### DISCUSSION THREAD

**Discussion by Jellyfrog** [community/community#4883] [Projects and Issues]
When having a textarea for logs or other long inputs, it would be good to be able to control rendered size of it, since it can become very long else.
Or maybe a option to mark a textarea as collapsed by default

**Comment by supermerill** [comment]
Also It could be nice to be able to display a big fat textarea when needed.

**Reply by MarketBeatAd** [reply]
Yes it would..

**Comment by MarketBeatAd** [comment]
I suggest we find a way to do this.
